assets/bindata_timestamp.txt

```diff
@@ -1 +1 @@
-1654679854
+1658914160
```

Comment:
Do we need to update this file manually? I thought it was automatically done using the bindata.sh script.

Reply:
No, sorry, my question wasn't clear. I see that this is changed. I expect that to only happen when the rebase script updates it as part of updating to a new release image, because the timestamp is supposed to come from the release file in the image. So, did this PR include a rebase? If so, maybe we can do that separately from enabling the new controller?